src/new-client/src/App/Footer/StatusButton/StatusButton.tsx

```diff
@@ -85,7 +86,7 @@ export const StatusButton: React.FC = () => {
   const url = "https://web-demo.adaptivecluster.com"
   const ref = useRef<HTMLDivElement>(null)
   const { displayMenu, setDisplayMenu } = usePopUpMenu(ref)
-  const services: ServiceStatus[] = []
+  const services: ServiceStatus[] = useServiceStatus()
 
   const toggleMenu = useCallback(() => {
```

Comment:
We shouldn't be using `useCallback` for things like this. Also, since the next value depends on the previous one: the toggle function should use the "function overload" of `setState`, like this: `() => { setDisplayMenu(prev => !prev) }` and I would much rather if that was defined directly in the `onClick` event-handle of `Button`.